metta/metta_io.py

```diff
@@ -117,14 +119,14 @@ def archive_matrix(
             raise IOError('Not a file: {}'.format(abs_path_file))
 
         if abs_path_file[-3:] == 'csv':
-            df_matrix = pd.read_csv(abs_path_file)
```

Comment:
Idea: to keep most of this code similar, is it possible to make a one-line dataframe here instead? I recalled that pandas.read_csv has a chunk size, and maybe it also has a row limit...

Just checked the docs. It looks like there is an nrows argument to read_csv, which I think we could use here.

Related: although the pandas read_hdf function has a far different interface, it looks it has something similar: start and stop. So maybe we could use that too?